File: Ambulance.BLL/Models/BrigadeMemberModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AmbulanceSystem.BLL.Models;

public class BrigadeMemberModel
{
    public int BrigadeMemberId { get; set; }
    public int PersonId { get; set; }
    public int BrigadeId { get; set; }
    public int RoleId { get; set; }
    public int MemberSpecializationTypeId { get; set; }
    public string? PersonFullName { get; set; }
    public string? BrigadeName { get; set; }
    public string? RoleName { get; set; }
    public string? SpecializationTypeName { get; set; }
}

File: ambulance-site/src/api/models/BrigadeMemberModel.js
export class BrigadeMemberModel {
    constructor({
        brigadeMemberId = 0,
        personId = 0,
        brigadeId = 0,
        roleId = 0,
        memberSpecializationTypeId = 0,
        personFullName = "",
        brigadeName = "",
        roleName = "",
        specializationTypeName = ""
    } = {}) {
        this.brigadeMemberId = brigadeMemberId;
        this.personId = personId;
        this.brigadeId = brigadeId;
        this.roleId = roleId;
        this.memberSpecializationTypeId = memberSpecializationTypeId;
        this.personFullName = personFullName;
        this.brigadeName = brigadeName;
        this.roleName = roleName;
        this.specializationTypeName = specializationTypeName;
    }
}